libs/core/src/ports/interfaces.ts

```diff
@@ -157,6 +157,87 @@ export interface Cache extends Disposable {
     key: string,
     ttlInSeconds: number,
   ): Promise<boolean>;
+
+  // Hash methods
+  incrementHashKey(
+    namespace: CacheNamespaces,
+    key: string,
+    field: string,
+    increment?: number,
+  ): Promise<number>;
+
+  getHash(
```

Comment:
It would be helpful to describe the new port interface here, especially given that the new functionality is not as trivial as basic key/value caching.